src/Ladybug_Shadow Study.py

```diff
@@ -117,33 +129,46 @@ def srfPtCalculator(i):
         else:
             for i in range(len(analysisSrfs)):
                 srfPtCalculator(i)
-    
+                
```

Comment:
@mostaphaRoudsari  I made those comments while I was reading the code. Not really necessary.